src/data/elements.js

```diff
@@ -5,25 +5,25 @@ module.exports = {
     demo: {
       primary: {
         js: [
-          '<button class="zep-button zep-button-primary">Default</button>',
-          '<button class="zep-button zep-button-primary zep-button--medium">Medium</button>',
-          '<button class="zep-button zep-button-primary zep-button--small">Small</button>',
-          '<button class="zep-button zep-button-primary" disabled>Disabled</button>',
-          '<button class="zep-button zep-button-primary zep-button-icon"><svg width="32" height="32" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-labelledby="title_copy" class="zep-button__icon"><title id="title_copy">Copy</title><path fill="currentColor" d="M20,21 L9,21 L9,7 L20,7 L20,21 Z M20,5 L9,5 C7.8954305,5 7,5.8954305 7,7 L7,21 C7,22.1045695 7.8954305,23 9,23 L20,23 C21.1045695,23 22,22.1045695 22,21 L22,7 C22,5.8954305 21.1045695,5 20,5 L20,5 Z M17,1 L5,1 C3.8954305,1 3,1.8954305 3,3 L3,17 L5,17 L5,3 L17,3 L17,1 Z"></path></svg></button>',
+          '<button class="zep-button zep-button--primary">Default</button>',
+          '<button class="zep-button zep-button--primary zep-button--medium">Medium</button>',
+          '<button class="zep-button zep-button--primary zep-button--small">Small</button>',
+          '<button class="zep-button zep-button--primary" disabled>Disabled</button>',
+          '<button class="zep-button zep-button--primary zep-button--icon"><svg width="32" height="32" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-labelledby="title_copy" class="zep-button__icon"><title id="title_copy">Copy</title><path fill="currentColor" d="M20,21 L9,21 L9,7 L20,7 L20,21 Z M20,5 L9,5 C7.8954305,5 7,5.8954305 7,7 L7,21 C7,22.1045695 7.8954305,23 9,23 L20,23 C21.1045695,23 22,22.1045695 22,21 L22,7 C22,5.8954305 21.1045695,5 20,5 L20,5 Z M17,1 L5,1 C3.8954305,1 3,1.8954305 3,3 L3,17 L5,17 L5,3 L17,3 L17,1 Z"></path></svg></button>',
```

Comment:
SVG is an old one. Please use the latest from our icon lib. ...also on following buttons